core/src/main/resources/error/error-classes.json

```diff
@@ -1834,6 +1849,11 @@
     ],
     "sqlState" : "42K05"
   },
+  "RESOLVED_ATTRIBUTE_MISSING_FROM_INPUT" : {
+    "message" : [
+      "<msg>"
```

Comment:
This is a red flag. Please, move the message out from source code. Imagine, we will translate this JSON to different languages.

Reply:
So we need change it as an internal error ?

Reply:
Consider to replace the messages https://github.com/apache/spark/blob/898ad77900d887ac64800a616bd382def816eea6/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala#L646-L660
by a sub-class of `UNRESOLVED_COLUMN` or add new ones.

Reply:
Got it.